apps/elasticsearch.go

```diff
@@ -156,72 +155,95 @@ func (LoggingProcessorMacroElasticsearchJson) Type() string {
 	return "elasticsearch_json"
 }
 
+func (p LoggingProcessorMacroElasticsearchJson) loggingProcessorElasticsearchJsonMultilineRules() []confgenerator.MultilineRule {
```

Comment:
Nit : Please update the method name `loggingProcessorElasticsearchJsonMultilineRules` to `parseMultilineRegex` and return the whole `LoggingProcessorParseMultilineRegex` processor instead of only the `MultilineRule`.